services/auth/token/helpers/token.js

```diff
@@ -0,0 +1,18 @@
+import to from 'await-to-js';
+import jwt from 'jsonwebtoken';
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+
+const SECRET_KEY = 'secretKeey';
+
+export const signToken = jsonToSign => {
```

Comment:
signToken = jsonToSign => {...
signToken = (jsonToSign) => {...
according to my linting rules

Reply:
What, i would say that no parameters is the way if there is only one argument passed down.